packages/core/src/lambda/commands/deploySamApplication.ts

```diff
@@ -0,0 +1,326 @@
+/*!
```

Comment:
is this file related somehow?

Reply:
looks like some more cases got merged on that last master merge that I missed. Should be resolved now.

Reply:
Oh weird, it looks like its still here. Maybe github UI is glitched? I think the PR is ready to be merged other than that

Reply:
Try `git rm <filepath>` on this file then push to PR.